packages/paypal-commerce-integration/src/paypal-commerce-ratepay/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -192,7 +192,7 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
         const formattedDate =
             date.getDate() < 10 ? `0${date.getDate()}` : date.getDate().toString();
         const formattedMonth =
-            date.getMonth() < 10 ? `0${date.getMonth() + 1}` : date.getMonth().toString();
+            date.getMonth() < 9 ? `0${date.getMonth() + 1}` : (date.getMonth() + 1).toString();
```

Comment:
It might be easier to create a simple method that checks the length of the string and adds 0, if there more that one character (or number in a string).

Also you can add the same behaviour for the `formattedDate`.

```
const formatDate = (date: number): string => {
    return `${date > 9 ? '' : 0}${date}`;
}

const formattedDate = formatDate(date.getDate());
const formattedMonth = formatDate(date.getMonth() + 1);
```

Reply:
Cover this code with tests please